packages/js/src/settings/helpers/validation.js

```diff
@@ -29,6 +33,23 @@ addMethod( number, "isMediaTypeImage", function() {
 	);
 } );
 
+addMethod( number, "isMediaMimeTypeAllowed", function() {
+	return this.test(
+		"isMediaMimeTypeAllowed",
+		__( "The selected media type is not valid. Supported types are: JPG, PNG, WEBP and GIF.", "wordpress-seo" ),
+		input => {
+			const media = select( STORE_NAME ).selectMediaById( input );
+
+			// No metadata to validate: default to valid.
+			if ( ! media ) {
+				return true;
+			}
+
+			return allowedMediaTypes.includes( media.mime_type );
```

Comment:
Sorry that I didn't notice this in CR, but we actually convert this to `mime` in the reducer: https://github.com/Yoast/wordpress-seo/blob/6681101100be1b3bb21667a3812aab548312224c/packages/js/src/settings/store/media.js#L53

In other words, I'm now always getting the warning 😅 
Maybe include a happy path in the test instructions too? hehe
```suggestion
			return allowedMediaTypes.includes( media.mime );
```